src/xcode/ENA.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -28,6 +37,24 @@
           "version": "0.12.0"
         }
       },
+      {
+        "package": "PathKit",
+        "repositoryURL": "https://github.com/kylef/PathKit.git",
+        "state": {
+          "branch": null,
+          "revision": "73f8e9dca9b7a3078cb79128217dc8f2e585a511",
+          "version": "1.0.0"
+        }
+      },
+      {
+        "package": "Spectre",
+        "repositoryURL": "https://github.com/kylef/Spectre.git",
+        "state": {
+          "branch": null,
+          "revision": "f79d4ecbf8bc4e1579fbd86c3e1d652fb6876c53",
+          "version": "0.9.2"
+        }
+      },
```

Comment:
are these two packages in the JSONSchema package included? Do we need to mention them in the licenses, too?

Reply:
No, I checked that with Marcus, its not needed.